.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
     - TASK=r_test
     # python package test
     - TASK=python_test
+    - OMP_NUM_THREADS=3 TASK=python_omp_test
```

Comment:
I think we can simply set nthread parameter in the specific test, and remove this execution group